Makefile

```diff
@@ -1,24 +1,23 @@
 .PHONY: requirements
 
 requirements: requirements.js
-	pip install -q -r requirements/base.txt --exists-action w
+	pip install .
 
 requirements.js:
 	npm install
 
-requirements.test: requirements
-	pip install -q -r requirements/test.txt --exists-action w
-
 install: requirements
 
-develop: requirements.test
+develop: requirements
+	pip install -U pytest>=2.7 pytest-mock pycodestyle pylint edx-lint==0.5.1
```

Comment:
Same comment here though....Groking your other feedback, I suppose what I'd prefer here is for `develop` to call the test requirements in pip install. I'm assuming that's done through `pip install .` but am less familiar with that convention so I lean on you for the right pattern. Ultimately though I don't want the list of libraries to be on this line, but stored somewhere else. (Which for now is setup.py)